chart/openfaas/README.md

```diff
@@ -70,12 +70,44 @@ helm repo update \
 
 > The above command will also update your helm repo to pull in any new releases.
 
+#### Tuning cold-start
+
+The concept of a cold-start in OpenFaaS only applies if you A) use faas-idler and B) set a specific function to scale to zero.
+
+There are two ways to reduce the Kubernetes cold-start for a pre-pulled image, which is 1-2 seconds.
+
+1) Don't set the function to scale down to zero, just set it a minimum availability i.e. 1/1 replicas
+2) Use async invocations via the `/async/function/<name>` route
+3) Tune the readinessProbes to be aggressively low values. This will reduce the cold-start at the cost of more `kubelet` CPU usage
+
+To achieve around 1s coldstart, set `values.yaml`:
+
+```yaml
+faasnetes:
+
+# redacted
+  readinessProbe:
+    initialDelaySeconds: 1
```

Comment:
The initial delay can be set to zero see https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/v1/defaults_test.go#L1611

Reply:
I have a recollection about setting one or more of these values to zero and them being automatically changed back to "1". I can't remember which one, but I thought it was this one.

Reply:
I've set it to zero on your recommendation in the "tuning" section.